package.json

```diff
@@ -25,6 +25,11 @@
   ],
   "contributes": {
     "commands": [
+      {
+        "command": "codesync.requestDemo",
+        "title": "Request a demo",
+        "category": "CodeSync"
+      },
       {
         "command": "codesync.signup",
         "title": "Authenticate",
```

Comment:
Maybe we can simply it to 2 options
1. Request a demo
2. Login

(Authenticate was to support both options, Login and SignUp. But since we don't have SignUp anymore, so maybe we should directly use Login instead of Authenticate)